src/gui/controllers/LoginController.java

```diff
@@ -19,7 +19,8 @@ public void initialize(URL arg0, ResourceBundle arg1) {
 	}
 
 	@FXML public void login(ActionEvent event) {
-		Utils.createWindow(null, LoginController.this, "../fxml/Home.fxml", "TasksFX", null, "../css/home.css");
+		ResourceBundle bundle = ResourceBundle.getBundle("resources.i18n.login");
```

Comment:
This should load the Home resources instead of login.
The line 23 is opening  the Home Window.